src/containers/markericon.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import L from 'leaflet';
+import Icon from '../components/icon';
+import ICONS from './icons';
+
+// Using L because a component returns React.createElement() instead of L.geoJSON(..., { pointToLayer }
+// Reference: https://github.com/PaulLeCam/react-leaflet/issues/234
+
+const setMarker = (properties, latlng) => {
+  const orderType = {
```

Comment:
refactor this to just return order specific properties (i.e., color) and then plug it into the marker in the base case.